.travis.yml

```diff
@@ -22,10 +22,10 @@ before_install:
   - cabal update && cabal install hlint hpc-coveralls
 
 script:
-  - ~/.cabal/bin/hlint .
+  - cd importer/ && ~/.cabal/bin/hlint . && cd ../
   - cd haskell-rpm/ && cabal install && cabal clean && cd ../
   - cd importer/ && cabal install --dependencies-only --enable-tests &&
-    cabal configure --enable-tests --enable-coverage --enable-library-coverage &&
```

Comment:
@bcl I think removing the --enable-library-coverage will break hpc-coveralls but let's see. I know this option is deprecated but when I was trying to enable coveralls that seemed to work for some reason.